week-1/implement/max.test.js

```diff
@@ -26,3 +26,29 @@ test.todo("given an empty array, returns -Infinity");
 // Given an array with non-number values
 // When passed to the max function
 // Then it should return the max and ignore non-numeric values
+
+const getMax = require("./max.js");
+
+describe("getMax", () => {
+  test("returns infinity for empty array", () => {
+    expect(getMax([])).toBe(-Infinity);
+  });
+
+  test("returns the number in the array if there's 1 number", () => {
+    expect(getMax([5])).toBe(5);
+    expect(getMax([8])).toBe(8);
+  })
+
+  test("returns the positive max number in the array", () => {
+    expect(getMax([5, 9, -4, -1, 2])).toBe(9);
+  })  
+
+  test("returns the largest decimal number in the array", () => {
+    expect(getMax([0.5, 0.9, 0.4, 0.1, 0.2])).toBe(0.9);
+  })  
+
+  test("returns the largest number in array with numerical and non-numerical value", () => {
+    expect(getMax([3, 7, 'hi', 1, 'bye'])).toBe(7);
+  }) 
```

Comment:
Can you try adding another case that has numbers are string in the array and see if your functionality still works? 
For example `["55", 8, 9]` Will this still work and give you the result of 9 not 55 cause 55 is a string?

Reply:
I have added that test case and changed my function accordingly.